components/Profile/Orders/Card.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <BaseCard :title="title">
+    <div v-if="orders.length">
+      <div
+        v-for="(order, index) in orders"
+        :key="order.id"
+        class="min-h-[100px] border-gray-300 pt-2.5 first:pt-0 last:pb-0"
+        :class="{
+          'border-b pb-2.5': index !== orders.length - 1,
+        }"
```

Comment:
I think it should work also like this:

```js
class="min-h-[100px] border-b border-gray-300 py-2.5 first:pt-0 last:border-b-0 last:pb-0"
```

I moved `border-b pb-2.5` and added `last:border-b-0` so border gets canceled on the last child.

Reply:
Done

Reply:
> Left some tailwind comments. Also we have TS issues: <img alt="image" width="832" src="https://user-images.githubusercontent.com/66744401/226378174-60c7adcb-c3bd-46b3-8008-9150938cb38a.png">

About Eslint issues, I've fixed it too